// Copyright 2018 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package cryptohome;

// Credential metadata structure used by FakeLECredentialBackend.
message FakeLECredentialMetadata {
  // Label used to store credential in the hash tree.
  optional uint64 label = 1;
  // Low Entropy secret.
  optional bytes le_secret = 2;
  // High Entropy secret.
  optional bytes he_secret = 3;
  // Reset secret.
  optional bytes reset_secret = 4;
  // Number of incorrect check attempts made.
  optional uint32 attempt_count = 5;
  // Valid PCR digest. For simplicity it's kept only for a single user.
  optional bytes valid_pcr_digest = 6;
}
